CHANGELOG/CHANGELOG-3.4.md

```diff
@@ -8,13 +8,18 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 
 ### etcd server
 - Fix [performance regression issue caused by the `ensureLeadership` in lease renew](https://github.com/etcd-io/etcd/pull/18440).
+- [keep tombstone if revision == compactAtRev](https://github.com/etcd-io/etcd/pull/18475)
 
 ### Package clientv3
 - [Print gRPC metadata in guaranteed order using the official go fmt pkg](https://github.com/etcd-io/etcd/pull/18311).
 
 ### Dependencies
 - Compile binaries using go [1.21.13](https://github.com/etcd-io/etcd/pull/18422).
 
+### Others
```

Comment:
It's not common for us to add changelog entries for tests. Is there particular reason we should call out these two changes to users via CHANGELOG?

Reply:
Hi @jmhbnz I added the tests because they're related to https://github.com/etcd-io/etcd/pull/18274. I can't just backport https://github.com/etcd-io/etcd/pull/18274 without these tests. However, currently the changelog only mentions functional change. I can remove the items anyway :p

Reply:
Right, usually we don't update changelog for test. Please consider removing it.